chain-tx-enclave-next/mls/src/group.rs

```diff
@@ -179,24 +181,28 @@ impl<CS: CipherSuite + Ord> GroupAux<CS> {
         })
     }
 
-    fn get_init_confirmed_transcript_hash(
-        &self,
-        sender: Sender,
-        commit: &Commit<CS>,
-    ) -> HashValue<CS> {
-        let interim_transcript_hash = b"".to_vec(); // TODO
+    /// spec: draft-ietf-mls-protocol.md#group-state
+    fn new_confirmed_transcript_hash(&self, sender: Sender, commit: &Commit<CS>) -> HashValue<CS> {
         let content_to_commit = message::MLSPlaintextCommitContent::new(
             self.context.group_id.clone(),
             self.context.epoch,
             sender,
             commit.clone(),
         )
         .get_encoding();
-        let to_hash = [interim_transcript_hash, content_to_commit].concat();
+        let to_hash = [
+            self.interim_transcript_hash
+                .as_ref()
+                .map(|h| h.as_ref())
+                .unwrap_or_default(),
```

Comment:
maybe worth expanding on what is going on here / why it matches the spec -- the type is (`&Vec<u8>.as_ref()`) `&[u8]` and its default == `b""` ? is that correct?

Reply:
yes, changed to more explicit `unwrap_or(b"")`, and copied the spec as comment.